sync/strategies/simple_onchange.py

```diff
@@ -3,7 +3,8 @@
 
 
 class SimpleOnChangeStrategy(IStrategy):
-    last_seen = {}
+    last_seen = {}  # store synchronized received memory & keep track of memory we have sent
```

Comment:
i think i'd prefer these to be defined in __init__

Reply:
sure